server/routes/index.js

```diff
@@ -1,82 +0,0 @@
-var express = require('express');
```

Comment:
Not deleted, but moved to `server/src/routes/index.js` (with Notion logic moved to `server/src/services/notion.service.js`)